sdv/datasets/demo.py

```diff
@@ -104,15 +104,19 @@ def _get_data(modality, output_folder_name, in_memory_directory):
     return data
 
 
-def _get_metadata(modality, output_folder_name, in_memory_directory):
-    metadata = MultiTableMetadata() if modality == 'multi_table' else SingleTableMetadata()
+def _get_metadata(output_folder_name, in_memory_directory, dataset_name):
+    metadata = Metadata()
     if output_folder_name:
         metadata_path = os.path.join(output_folder_name, METADATA_FILENAME)
-        metadata = metadata.load_from_json(metadata_path)
+        metadata = metadata.load_from_json(metadata_path, dataset_name)
```

Comment:
Is passing the `dataset_name` here just a precaution? It shouldn't be required here because there shouldn't ever be a case where the json is in single table format right?

Reply:
Yeah it's a precaution, not necessary and it defaults to None. For multi-table it does not affect anything